code-challenges.js

```diff
@@ -31,6 +45,25 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+//creating a function named boil that tales the parameter num
+//const boil = (num) => {
+//   //if the num is 212 then it will return 212
+//   // is at boiling point
+//   if (num === temperature3){
+//     return `${num} is at boiling point.`
+//     // if num less than 212  it will
+//     //return below boiling point
+//   } else if (num<temperature3){
+//     return `${num} is below boiling point`
+//     //everything else will be above biling point
+//   } else {
+//     return `${num} is above boiling point`
+//   }
+// }
+// console.log(boil(temperature1)) ---> 42 is below boiling point
```

Comment:
I love the fact that you took advantage of the variable provided which happens to be at boiling point. Just keep in mind that it would be best if we designated that variable in our function ourselves because temperature3 is a test variable that simulates an input. Great job!